Pipfile.lock

```diff
@@ -312,84 +320,6 @@
             "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
             "version": "==1.16.0"
         },
-        "ujson": {
```

Comment:
Looks like you may need to run `pipenv lock` again. ujson and uvloop are speedup dependencies for sanic—not sure why they are removed.

Reply:
I have finally managd to get these fixed with a relock. Not sure what went different this time but it magically worked. Can you review again?